ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/widgets/editortab/EditorTabWidget.ui.xml

```diff
@@ -39,6 +41,13 @@
             color: tabTextColor;
             cursor: default;
             margin-bottom: 1px;
+
+            -webkit-transition: all 0.3s ease-in-out,
+                                fill 0.3s ease-in-out,
```

Comment:
Below here is simple transition declaration:
```
transition: all 0.3s ease-in-out
```

And according to this reference http://www.w3schools.com/css/css3_transitions.asp it should